integreat_cms/static/src/js/dashboard/broken-links.ts

```diff
@@ -16,7 +16,20 @@ const getContent = async (url: string): Promise<Content> => {
     return response.json();
 };
 
+const showAllTotalNumbers = () => {
+    const elements = document.querySelectorAll<HTMLElement>(".total-results");
+
+    elements.forEach((element) => {
+        if (!element.closest("#broken-links")) {
+            const el = element;
+            el.classList.remove("hidden");
+        }
+    });
+};
+
 window.addEventListener("load", async () => {
+    showAllTotalNumbers();
+
```

Comment:
Wouldn't it make much more sense to change the definition of `updateNumberOfPages()` below instead? It is the one updating the total once the data has been retrieved. Putting `showAllTotalNumbers()` at the top of the `load` event listener instead removes the class `hidden` on page load, before we even start fetching the auxiliary data.

I would propose to entirely remove the definition of `showAllTotalNumbers()` in favour of changing `updateNomberOfPages()` as follows:

```diff
     const updateNumberOfPages = (numberOfPages: number) => {
-        (brokenLinksElement.querySelector(".total-results span") as HTMLElement).innerText = numberOfPages.toString();
+        const results = brokenLinksElement.querySelector(".total-results") as HTMLElement;
+        (results.querySelector("span") as HTMLElement).innerText = numberOfPages.toString();
+        results.classList.remove("hidden");
     };
```

Reply:
This suggestion is invalid and I have not read the originally suggested changes attentively enough